config/elasticsearch.yml

```diff
@@ -20,7 +20,7 @@ http:
 
 searchguard:
     ssl.transport:
-        enabled: true
```

Comment:
I tried that already and search-guard-ssl expect this value to be `true`

it seem that search-guard-ssl must be... not installed